### ISSUE TITLE
May I ask the config parameters of your best result model？

### ISSUE THREAD

**Issue by ligungun** [Dw1010/IntagHand#17]
Thank you very much for making your training code public. I used your default config file to train the model. The best results achieved 9.5 for MPJPE and 9.8 for MPVPE. May I ask what is the config parameter that you get the best model training. Looking forward to your reply. Thanks.

**Comment by hanhuili** [comment]
> Thank you very much for making your training code public. I used your default config file to train the model. The best results achieved 9.5 for MPJPE and 9.8 for MPVPE. May I ask what is the config parameter that you get the best model training. Looking forward to your reply. Thanks.

I also wonder whether the published training code is really the one used for training? because I notice the gt j2d is not aligned with the image, e.g.,
![image](https://user-images.githubusercontent.com/22837398/218364011-90120272-6edb-4517-9975-4725f858c2f9.png)

**Comment by ligungun** [comment]
> 
Is this the joint projection of your single hand image? I haven't rendered it. Have you trained with the shared training code? Do you get the same result as me?

**Comment by ligungun** [comment]
> > Thank you very much for making your training code public. I used your default config file to train the model. The best results achieved 9.5 for MPJPE and 9.8 for MPVPE. May I ask what is the config parameter that you get the best model training. Looking forward to your reply. Thanks.
> 
> I also wonder whether the published training code is really the one used for training? because I notice the gt j2d is not aligned with the image, e.g., ![image](https://user-images.githubusercontent.com/22837398/218364011-90120272-6edb-4517-9975-4725f858c2f9.png)

**Comment by hanhuili** [comment]
> > 
> 
> Is this the joint projection of your single hand image? I haven't rendered it. Have you trained with the shared training code? Do you get the same result as me?

No, this is the rendered gt, I haven't trained the model. How long does it take to train the model and what kind of hardware you are using?

**Comment by ligungun** [comment]
> > Is this the joint projection of your single hand image? I haven't rendered it. Have you trained with the shared training code? Do you get the same result as me?
> 
> No, this is the rendered gt, I haven't trained the model. How long does it take to train the model and what kind of hardware you are using?

I observed some offset between the gt mesh and the position of the hand in the image, but it shouldn't deviate so much. Have you looked at the rendered results of the gt in the other pictures? If they are all like this, there may be a problem with the data. It took me 4.5 days to train the model with four 3090 GPUs.

**Comment by hanhuili** [comment]
> > > Is this the joint projection of your single hand image? I haven't rendered it. Have you trained with the shared training code? Do you get the same result as me?
> > 
> > 
> > No, this is the rendered gt, I haven't trained the model. How long does it take to train the model and what kind of hardware you are using?
> 
> I observed some offset between the gt mesh and the position of the hand in the image, but it shouldn't deviate so much. Have you looked at the rendered results of the gt in the other pictures? If they are all like this, there may be a problem with the data. It took me 4.5 days to train the model with four 3090 GPUs.

Other images have the same problem, but I have modified the code and I wonder if this problem can be ignored...

**Comment by adwardlee** [comment]
> Thank you very much for making your training code public. I used your default config file to train the model. The best results achieved 9.5 for MPJPE and 9.8 for MPVPE. May I ask what is the config parameter that you get the best model training. Looking forward to your reply. Thanks.

Hi, @ligungun, did you change the default parameters in the defaults.yaml? I can only reproduce MPJPE 10.4mm with the default settings. Thanks.

**Comment by ligungun** [comment]
> > Thank you very much for making your training code public. I used your default config file to train the model. The best results achieved 9.5 for MPJPE and 9.8 for MPVPE. May I ask what is the config parameter that you get the best model training. Looking forward to your reply. Thanks.
> 
> Hi, @ligungun, did you change the default parameters in the defaults.yaml? I can only reproduce MPJPE 10.4mm with the default settings. Thanks.

I did not change the default parameters. The code has not changed anywhere. It is possible that the model optimization is unstable. Or it may be related to the number of GPUs you use for training. I observed that when using the ZeroRedundancyOptimizer optimizer, the size of the saved model is related to the number of GPUs used for training. The larger the number of GPUs, the larger the saved model. The size of the saved model is the same as the model published by the author when only one GPU is used. When I change the optimizer to torch.optim.Adam, no matter how many GPUs are used for training, the saved model size is the same as the model size released by the author. But I don't know if this will affect the test results.

**Comment by Rhett009** [comment]
Have you done the ablation study

**Comment by xungeer29** [comment]
> Thank you very much for making your training code public. I used your default config file to train the model. The best results achieved 9.5 for MPJPE and 9.8 for MPVPE. May I ask what is the config parameter that you get the best model training. Looking forward to your reply. Thanks.

The result I obtained using the default parameters is 9.6(MPVPE) and 9.788(MPVPE).
single 3090
